utils.go

```diff
@@ -85,7 +85,13 @@ func doDeposit(engine *core.Engine, payment *common.Payment,
 	return
 }
 
+// getUserIDFromAccount extracts user ID string from account with
+// system prefix and parses it to uint32.
 func getUserIDFromAccount(account string) (uint32, error) {
+	parts := strings.Split(account, "_")
+	if len(parts) == 2 {
+		account = strings.TrimPrefix(account, "exchange_")
```

Comment:
What if we add another prefix? Why don't just remove any part before "_"?

Reply:
This changes are outdated, will remove this. If you remember at first we tried to make one connector for zigzag and exchange. And then we decided to make two connectors. This prefix for account do needed now.